app/Http/Controllers/topicController.php

```diff
@@ -9,23 +9,35 @@ class topicController extends Controller
 {
     //
     public function getList() {
-    	return view('admin.topic.list');
+        $Topic =topic::all();
+        return view('admin.topic.list',['Topic'=>$Topic]);
     }
 
     public function getCreate() {
     	return view('admin.topic.create');
     }
 
-    public function postCreate() {
-
+    public function postCreate(Request $request) {
+        $createTopic=new topic;
+        $createTopic->nametopic=$request->createNameTopic;
+        $createTopic->ansinametopic=$request->createAnsinameTopic;
+        $createTopic->save();
+        return redirect('admin/topic/list')->with('notify','Create successful : '.$request->createNameTopic);
     }
 
-    public function getUpdate() {
-    	return view('admin.topic.update');
+    public function postUpdate(Request $request, $idtopic) {
+        $updateTopic = topic::find($idtopic);
+        $updateTopic->nametopic=$request->updateNameTopic;
+        $updateTopic->ansinametopic=$request->updateansiNameTopic;
+        $updateTopic->save();
+        return redirect('admin/topic/list')->with('notify','Update successful : '.$request->updateNameTopic); 
     }
 
-    public function postUpdate() {
-        
+    public function getDelete($idtopic){
+        $deleteTopic= topic::find($idtopic);
+        $nameTopic=$deleteTopic->nametopic;
```

Comment:
chưa hiểu cái này để làm gì?

Reply:
$nametopic để nối thêm vào cái đoạn thông báo  return redirect() thôi. Vd: " Delete successful : Toàn Chó "